lib/cfme/versions/version.rb

```diff
@@ -0,0 +1,5 @@
+module CFME
+  class Versions
+    GEM_VERSION = "0.1.0"
```

Comment:
I intentionally designed this project to be a single script without a lib dir so that if someone wanted to vendor this, they could and it was easy.  If someone wanted to just copy the single file into `/usr/local/bin`, they could and it was easy.

Breaking this up has zero value to this project as what I had before was still allowed to be digested by the `.gemspec`, which is the only really good reason I could think of for breaking this up.

Also, there is as `cfme-versions --version` flag that is meant to view the version of the script, and now this doesn't match.  If you have it read in this file to now sync those two up, you break the ability for this to be a single script as originally intended.